lib/components/molecules/Accordian/Accordian.style.js

```diff
@@ -0,0 +1,156 @@
+import { css } from 'styled-components';
+import 'react-accessible-accordion/dist/fancy-example.css';
```

Comment:
@naisergic even after writing so much of CSS below you still need to import CSS from this package?